src/main/java/org/folio/circulation/resources/RequestCollectionResource.java

```diff
@@ -104,7 +104,7 @@ void replace(RoutingContext routingContext) {
 
     final RequestFromRepresentationService requestFromRepresentationService =
       new RequestFromRepresentationService(
-        new ItemRepository(clients, false, true, true),
+        new ItemRepository(clients, true, true, true),
```

Comment:
Please add the required permissions to the relevant endpoint in the module descriptor to allow these additional HTTP requests to be made.

Reply:
Added corresponding permissions for all endpoints which uses `ItemRepository` with `fetchLocation` flag.